src/uniprotkb/components/statistics/HistoricalReleasesEntriesLinePlot.tsx

```diff
@@ -0,0 +1,113 @@
+import { memo, useCallback, useEffect, useRef } from 'react';
+import {
+  select,
+  scaleLinear,
+  axisBottom,
+  axisLeft,
+  line,
+  scaleTime,
+  format,
+  timeFormat,
+} from 'd3';
+
+import styles from './styles/historical-release-entries-line-plot.module.scss';
+
+// Specify the chart’s dimensions.
+const width = 400;
+const height = 400;
+const margin = { top: 20, right: 60, bottom: 45, left: 80 };
+
+export type DateCount = [Date, number];
+export type Bounds = {
+  date: [Date, Date];
+  count: [number, number];
+};
+
+type Props = {
+  dateCounts?: DateCount[];
+  bounds?: Bounds;
+};
+
+const HistoricalReleasesEntriesLinePlot = ({ dateCounts, bounds }: Props) => {
+  const svgRef = useRef<SVGSVGElement>(null);
+
+  const renderPlot = useCallback((dateCounts: DateCount[], bounds: Bounds) => {
+    const chart = select(svgRef.current).select('g');
+
+    // x-axis
+    const xScale = scaleTime()
+      .domain(bounds.date) // units: Date
+      .range([0, width]); // units: pixels
+    chart
+      .select<SVGGElement>('.x-axis')
+      .transition()
+      .duration(1_000)
+      .call(
+        axisBottom(xScale).tickFormat((d: Date | { valueOf(): number }) =>
+          timeFormat('%Y')(new Date(d.valueOf()))
+        )
+      );
+
+    // y-axis
+    const yScale = scaleLinear()
+      .domain(bounds.count) // units: count
+      .range([height, 0]); // units: pixels
```

Comment:
```suggestion
      .range([height, 0]) // units: pixels
      .nice();
```
Make sure that we have a bit of margin at the top so that the highest number is not too close to the top and that the last tick (at, or higher than, the highest value) will be displayed.